doc/source/changes/version_0_32.rst.inc

```diff
@@ -55,4 +55,11 @@ Miscellaneous improvements
 Fixes
 ^^^^^
 
-* fixed something (closes :issue:`1`).
+* fixed reading/exporting sessions containing two or more axes/groups
```

Comment:
The work you have put into this PR makes me wonder even more whether *saving* sessions (and especially when using non LArray objects) to CSV and to Excel,  are worth it. I fear they will give us an endless stream of problems for little benefit.

Reply:
No one noticed the bug meaning no one will notice if we drop the ability to save and load axes and groups with the CSV or Excel format. 
So, do I edit the title of the corresponding issue and drop Axis and Group objects when calling Session.save()/load() ?

Reply:
I am unsure where to draw the line/what's best:
1) support saving axes and groups (and possibly arrays with metadata) **only** to HDF (or other formats where we can store arbitrary things). In that case, we will need to output a warning when saving a session containing them to csv/excel. Or we could go the GIMP route and say there is a single "native" format (HDF or .la or whatever) and there are other formats we can export to, each with their own limitations.
2) only support saving sessions to HDF-like formats.
3) keep trying to support everything for all formats. Let's be clear: it would love to have that, but I fear the time spent on that could be better spent elsewhere.
4) possibly another intermediate option I am not seeing?
I think we need to discuss this face to face, as it will be easier.